junit_alcotest 2.2.0 fails to build on OCaml 5.2.1

When trying to install junit_alcotest 2.2.0 we get this error:
```
[ERROR] The compilation of junit_alcotest.2.2.0 failed at "dune build -p junit_alcotest -j 1 @install".

#=== ERROR while compiling junit_alcotest.2.2.0 ===============================#
# context     2.3.0 | linux/x86_64 | ocaml-base-compiler.5.2.1 | git+https://github.com/ocaml/opam-repository.git
# path        ~/work/semgrep-proprietary/semgrep-proprietary/_opam/.opam-switch/build/junit_alcotest.2.2.0
# command     ~/work/semgrep-proprietary/semgrep-proprietary/_opam/bin/dune build -p junit_alcotest -j 1 @install
# exit-code   1
# env-file    ~/.opam/log/junit_alcotest-33361-9f770a.env
# output-file ~/.opam/log/junit_alcotest-33361-9f770a.out
### output ###
# (cd _build/default && /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/bin/ocamlopt.opt -w -40 -g -I alcotest/.junit_alcotest.objs/byte -I alcotest/.junit_alcotest.objs/native -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/alcotest -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/alcotest/engine -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/alcotest/stdlib_ext -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/astring -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/cmdliner -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/fmt -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/fmt/cli -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/fmt/tty -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/junit -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/ocaml/unix -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/ptime -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/ptime/clock -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/re -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/seq -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/stdlib-shims -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/tyxml -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/tyxml/functor -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/uutf -intf-suffix .ml -no-alias-deps -o alcotest/.junit_alcotest.objs/native/junit_alcotest.cmx -c -impl alcotest/junit_alcotest.ml)
# File "alcotest/junit_alcotest.ml", line 35, characters 6-32:
# 35 |     | Alcotest_engine__Core.Skip as exn ->
#            ^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound constructor Alcotest_engine__Core.Skip
# (cd _build/default && /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/bin/ocamlc.opt -w -40 -g -bin-annot -bin-annot-occurrences -I alcotest/.junit_alcotest.objs/byte -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/alcotest -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/alcotest/engine -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/alcotest/stdlib_ext -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/astring -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/cmdliner -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/fmt -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/fmt/cli -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/fmt/tty -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/junit -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/ocaml/unix -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/ptime -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/ptime/clock -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/re -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/seq -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/stdlib-shims -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/tyxml -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/tyxml/functor -I /home/runner/work/semgrep-proprietary/semgrep-proprietary/_opam/lib/uutf -intf-suffix .ml -no-alias-deps -o alcotest/.junit_alcotest.objs/byte/junit_alcotest.cmo -c -impl alcotest/junit_alcotest.ml)
# File "alcotest/junit_alcotest.ml", line 35, characters 6-32:
# 35 |     | Alcotest_engine__Core.Skip as exn ->
#            ^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound constructor Alcotest_engine__Core.Skip
```

--- a/alcotest/junit_alcotest.ml
+++ b/alcotest/junit_alcotest.ml
@@ -32,7 +32,7 @@ let wrap_test ?classname handle_result (name, s, test) =
         exn_msg
       |> handle_result;
       reraise exn
-    | Alcotest_engine__Core.Skip as exn ->
+    | Alcotest_engine.V1.Core.Skip as exn ->
       Junit.Testcase.skipped ~name ~classname ~time:0. |> handle_result;
       reraise exn
     | exn ->

--- a/junit_alcotest.opam
+++ b/junit_alcotest.opam
@@ -10,7 +10,7 @@ tags: ["junit" "jenkins" "alcotest"]
 depends: [
   "dune" {>= "3.0"}
   "odoc" {with-doc & >= "1.1.1"}
-  "alcotest" {>= "1.8.0"}
+  "alcotest" {>= "1.9.0"}
   "junit" {= version}
   "ocamlformat" {= "0.27.0" & with-dev-setup}
 ]

